torch/jit/_monkeytype_config.py

```diff
@@ -72,6 +72,11 @@ def consolidate_types(self, qualified_name: str) -> Dict:
                     if inspect.getmodule(_type) == typing:
                         _type_to_string = str(_type)
                         _all_type += _type_to_string.replace('typing.', '') + ','
+                    elif "torch.nn.parameter.Parameter" in str(_type):
```

Comment:
Interesting, what is the type of `_type`? Why is string comparison used instead of simple equality check?

If string comparison is indeed needed, why is `in` used for comparison?

Reply:
So printing `_type` -> `<class 'torch.nn.parameter.Parameter'>`
Printing `str(_type)` ->  `<class 'torch.nn.parameter.Parameter'>` 

I cannot use a direct string compare with `_type` nor `str(_type)`

Reply:
I mean, is it possible to use direct object equality check?
```
if _type is torch.nn.parameter.Parameter:
   xxxxx
```

Reply:
Done, can you check now.